apps/showcase/src/app/sdk-training/sdk-training.component.ts

```diff
@@ -0,0 +1,14 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {TrainingComponent} from '../../components';
+
+@Component({
+  selector: 'o3r-sdk-training',
+  standalone: true,
+  imports: [
+    TrainingComponent
+  ],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: '<o3r-training [title]="\'SDK Training\'" [trainingPath]="\'sdk-training\'"></o3r-training>',
+  styleUrls: ['./sdk-training.component.scss']
```

Comment:
why using `styleUrls` and not `styles` as you use `template` and not `templateUrl`

Reply:
the file contains several lines of style so for readability reasons, it is better to use `styleUrls`
for the template, we could use `templateUrl` but this file would contain only one line so I don't know if it would be worth it, what do you think?